src/Utils/Logging/Config.hs

```diff
@@ -197,7 +198,8 @@ validateBackendConfig o = do
         validateHandleConfig $ _backendConfigHandle o
         case (_backendConfigHandle o, _backendConfigColor o) of
             (FileHandle _, ColorTrue) ->
-                tell ["log messages are formatted using ANSI color escape codes but are written to a file"]
+              liftIO $ throwError
+              $ error "log messages are formatted using ANSI color escape codes but are written to a file"
```

Comment:
I think this `error` is unnecessary.

Reply:
Yeah, don't use `error` here.

``` Haskell
λ> :t error
error :: [Char] -> a
```

So, `error` is like `undefined`, or an infinite loop (`let a = a in a`). It can have any type, so it is a valid argument to each function. But's usually not what you want.